cmd/web3/main.go

```diff
@@ -978,6 +1008,49 @@ func Send(ctx context.Context, rpcURL, privateKey, toAddress, amount string) {
 	}
 	fmt.Println("Transaction address:", tx.Hash.Hex())
 }
+func GenerateCode(ctx context.Context, c *cli.Context) {
+	var abis []string
+	var bins []string
+	var types []string
+	var lang bind.Lang
+
+	switch c.String("lang") {
+	case "go":
+		lang = bind.LangGo
+	case "java":
+		lang = bind.LangJava
+	case "objc":
+		lang = bind.LangObjC
+	default:
+		fatalExit(fmt.Errorf("Unsupported destination language: %v", lang))
+	}
+
+	abiFile := c.String("abi")
+
+	if abiFile == "" {
+		fatalExit(errors.New("Please set the ABI file name"))
+	}
+
+	abi, err := ioutil.ReadFile(abiFile)
+	if err != nil {
+		fatalExit(fmt.Errorf("Failed to read file %q: %v", abiFile, err))
+	}
+
+	abis = append(abis, string(abi))
+	bins = append(bins, "")
+	types = append(types, c.String("pkg"))
```

Comment:
These three slices are sort of confusing. What about just declaring them here like:
```suggestion
	types := []string{c.String("pkg")}
```